src/System.Transactions/src/System/Transactions/CommittableTransaction.cs

```diff
@@ -42,6 +44,12 @@ internal CommittableTransaction(IsolationLevel isoLevel, TimeSpan timeout) : bas
             }
         }
 
+        // This serializtion constructor needs to be removed when we implement distributed transaction capabilities
```

Comment:
Nit: serializtion => serialization

Is this comment (and the others like it) necessary?  Since the ctor is private, seems like any changes that need to be made as part of adding distributed capabilities would be made, including removing this if removing this was necessary.

I'm curious what this would end up being replaced by, though.  Transaction implements ISerializable, which typically means the class needs to have a deserialization ctor to go along with the ISerializable.GetObjectData; the primary exception to that is if GetObjectData ends up setting a surrogate to be used for deserialization... is that the case here?